normandy/recipes/models.py

```diff
@@ -431,7 +431,6 @@ def _create_new_enabled_state(self, **kwargs):
         self.recipe.update_signature()
         self.recipe.save()
 
-    @transaction.atomic
```

Comment:
So that is enough? 
Shouldn't we catch and log Remote Settings client exceptions?
Django does not rollback DB operations if an error is raised and a 5XX is returned?

Reply:
Exceptions should be caught and logged by our normal error handling mechanisms (I don't know why that didn't happen in the case we saw this morning). I don't think we need any additional error handling besides what we already have.

Django's default behavior is to auto-commit every database call, so there is no request-level transaction that can be rolled back.